### PR TITLE
Refactor access to APIs and configs

### PR DESCRIPTION
**Pull request by philippQubit**
Some simplifications and a caching config layer

### CODE REVIEW

File: mlviews/sql_compiler.py [hunk 1]
```diff
@@ -1,49 +1,15 @@
-import requests
 import mlviews.template_manager as template_manager
```
Comment (thomas-woodruff) [inline mlviews/sql_compiler.py]: I don't think you need to import this anymore

File: mlviews/core.py [hunk 2]
```diff
@@ -4,27 +4,31 @@
...
+
 	# compile sql
-	sql = sql_compiler.compile_sql((nimbus_replacements, client_api_replacements, {'dataset_name': dataset_name}), template_id)
+	sql = sql_compiler.compile_sql(replacements, template)
 
 	return sql
 
 
-def compile_and_create_or_replace_view(tracking_id, template_id, client):
+def compile_and_create_or_replace_view(tracking_id, template_id, client, config_map):
```
Comment (thomas-woodruff) [inline mlviews/core.py]: I think you may need to pass config_map into `compile_sql` within this function

Comment (philippQubit) [reply mlviews/core.py]: Addressed

File: mlviews/sql_compiler.py (hunk 1 above)
Comment (philippQubit) [reply mlviews/sql_compiler.py]: Done...

**Comment by philippQubit** [comment]
Yes - thx for the help setting up the credentials. Integration tests are passing

**Pull request merged**